### ISSUE TITLE
stratis-cli requires python-dbus-client-gen at least 0.6 but only 0.5 is specified in setup.py

### ISSUE THREAD

**Issue by mulkieran** [stratis-storage/stratis-cli#288]
python-dbus-client-gen's latest version is 0.7. Verify that current version of stratis-cli does not require that version.

**Comment by mulkieran** [comment]
If running against python-dbus-client-gen v0.5, the following:

```
$ PYTHONPATH=../dbus-python-client-gen/mystuff:./src ./bin/stratis --help
Traceback (most recent call last):
  File "./bin/stratis", line 24, in <module>
    from stratis_cli import run
  File "/home/mulhern/stratis2/stratis-cli/src/stratis_cli/__init__.py", line 18, in <module>
    from ._main import run
  File "/home/mulhern/stratis2/stratis-cli/src/stratis_cli/_main.py", line 27, in <module>
    from ._error_reporting import handle_error
  File "/home/mulhern/stratis2/stratis-cli/src/stratis_cli/_error_reporting.py", line 28, in <module>
    from ._actions import interface_name_to_common_name
  File "/home/mulhern/stratis2/stratis-cli/src/stratis_cli/_actions/__init__.py", line 18, in <module>
    from ._data import interface_name_to_common_name
  File "/home/mulhern/stratis2/stratis-cli/src/stratis_cli/_actions/_data.py", line 231, in <module>
    DBUS_TIMEOUT_SECONDS)
TypeError: make_class() takes 2 positional arguments but 3 were given
```

**Comment by mulkieran** [comment]
>= 0.6 is the requirement. dbus-python-client-gen 0.7 extended the error heirarchy and exported a few more error types in its public interface, but stratis-cli does not use these names explicitly. The associated error messages are a nice thing to have, but not required.

**Comment by mulkieran** [comment]
Resolved by #289 .
